src/styles.js

```diff
@@ -47,7 +58,7 @@ export function loaders(options) {
 export function plugins(options) {
   return options.useDevServer
     ? []
-    : [new ExtractTextPlugin({
-        filename: options.useHashedAssetNames ? 'app.[contenthash].css' : 'app.css', 
-        allChunks: true})]
+    : [new MiniCssExtractPlugin({
+      filename: options.useHashedAssetNames ? 'app.[contenthash].css' : 'app.css',
+    })]
```

Comment:
Please add OptimizeCssAssetsPlugin in production - it should take care of optimizing and minimizing css files after it got extracted from all modules. Example config:
```
        new OptimizeCssAssetsPlugin({
              assetNameRegExp: /\.css$/g,
              cssProcessor: cssnano,
              cssProcessorOptions: {
                discardComments: { removeAll: true },
                discardEmpty: true
              },
              canPrint: true
            })
```